TeamCode/src/main/java/org/firstinspires/ftc/teamcode/MainSystem.java

```diff
@@ -61,15 +93,19 @@ public void runOpMode() {
 
         waitForStart();
 
+
         while (opModeIsActive()) {
             CommandScheduler.getInstance().run();
 
+        waitForStart();
+
+        while (opModeIsActive()) {
```

Comment:
but why

Reply:
beacuse reasons and life sometimes is cruel im sorry alberto wont happen again i love u